TTPChallenge/AllTechTypesVC.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  AllTechTypesVC.swift
+//  TTPChallenge
+//
+//  Created by Mirim An on 2/15/17.
+//  Copyright © 2017 TeamMDC. All rights reserved.
+//
+
+import UIKit
+
+class AllTechTypesVC: UIViewController {
+
+    @IBOutlet var allTechJobsButtons: [UIButton]!
```

Comment:
You should avoid using tags. It's bad news bears. Alternatively, you can check the senders label. so your method would be '''func techTypeBtnTapped(button sender: UIButton)''' and you would check the '''button.label''' property, which is still a little gross, but you can turn the labels into enums. The safest way to do this is just do it in storyboard and connect the view to it's respective view controller via a segue. Additionally, you might want to consider de-cluttering your storyboard through the use of storyboard references. https://code.tutsplus.com/tutorials/ios-9-staying-organized-with-storyboard-references--cms-24226

http://doing-it-wrong.mikeweller.com/2012/08/youre-doing-it-wrong-4-uiview.html